### PR TITLE
Adjust to use new API for vt references.

### PR DESCRIPTION
**Pull request by janowagner**
The changes correspond to https://github.com/greenbone/gvm-libs/pull/225
where the C API is change to have unified cross references.

### CODE REVIEW

File: src/manage_sql_nvts.c [hunk 1]
```diff
@@ -206,10 +207,16 @@ make_nvt_from_nvti (const nvti_t *nvti)
   else
     chunk_count++;
 
+  cve = nvti_refs (nvti, "cve", "", 0);
+  bid = nvti_refs (nvti, "bid", "", 0);
+  xref = nvti_refs(nvti, NULL, "cve,bid", 1);
```
Comment (mattmundell) [inline src/manage_sql_nvts.c]: Missing space.

File: src/manage_sql.c [hunk 2]
```diff
@@ -24611,83 +24611,17 @@ result_iterator_nvt_xref (iterator_t *iterator)
...
-      item++;
-    }
-  g_strfreev (split);
-  g_free (str);
-
-  str = nvti_refs (nvti, NULL, "cve,bid", 1);
-  split = g_strsplit (str, ",", 0);
-  item = split;
-  while (*item)
+  for (i = 0;i < nvti_ref_len (nvti);i ++)
```
Comment (mattmundell) [inline src/manage_sql.c]: Missing spaces after the ;'s.  Extra space before the ++.

File: src/manage_sql.c [hunk 3]
```diff
@@ -24611,83 +24611,17 @@ result_iterator_nvt_xref (iterator_t *iterator)
...
-          continue;
-        }
-
-      split2 = g_strsplit (type_and_id, ":", 2);
-      if (split2[0] && split2[1])
-        xml_string_append (xml, "<ref type=\"%s\" id=\"%s\"/>", split2[0], split2[1]);
-
-      item++;
+      vtref_t *ref = nvti_ref (nvti, i);
+      xml_string_append (xml, "<ref type=\"%s\" id=\"%s\"/>", vtref_type(ref), vtref_id(ref));
```
Comment (mattmundell) [inline src/manage_sql.c]: Missing spaces before the ('s.

File: src/manage_migrators.c [hunk 4]
```diff
@@ -1701,6 +1738,7 @@ static migrator_t database_migrators[] = {
   {205, migrate_204_to_205}, // v8.0: rev 205
   {206, migrate_205_to_206},
   {207, migrate_206_to_207},
+  {208, migrate_207_to_208},
```
Comment (mattmundell) [inline src/manage_migrators.c]: GVMD_DATABASE_VERSION in CMakeLists.txt needs to be updated for this to run.

File: src/manage_sql_nvts.c (hunk 1 above)
Comment (janowagner) [reply src/manage_sql_nvts.c]: Fixed.

File: src/manage_sql.c (hunk 2 above)
Comment (janowagner) [reply src/manage_sql.c]: Fixed.

File: src/manage_sql.c (hunk 3 above)
Comment (janowagner) [reply src/manage_sql.c]: Fixed,

File: src/manage_migrators.c (hunk 4 above)
Comment (janowagner) [reply src/manage_migrators.c]: Fixed.

File: src/manage_sql.c (hunk 2 above)
Comment (mattmundell) [reply src/manage_sql.c]: There's still a space before the ++.

Comment (janowagner) [reply src/manage_sql.c]: Fixed.

**Pull request merged**